src/main/scala/com/raphtory/core/implementations/pojograph/entities/external/PojoExVertex.scala

```diff
@@ -45,6 +45,8 @@ class PojoExVertex(private val v: PojoVertex,
   def getOutEdge(id: Long,after:Long=0L,before:Long=Long.MaxValue): Option[Edge] = individualEdge(internalOutgoingEdges,after,before,id)
   //In edges individual
   def getInEdge(id: Long,after:Long=0L,before:Long=Long.MaxValue): Option[Edge]  = individualEdge(internalIncomingEdges,after,before,id)
+  // edge individual
+  def getEdge(id: Long,after:Long=0L,before:Long=Long.MaxValue): Option[Edge]  = individualEdge(internalIncomingEdges++internalOutgoingEdges,after,before,id)
```

Comment:
The only issue I could see here is that if there is an incoming and outgoing edge we may return different objects each time. If the user is using History etc this could be a problem. But as long as we give reasonable warning I think this is fine!